test/unit/test_il_definitions.c

```diff
@@ -17,483 +17,483 @@ bool test_rzil_bv_init32(void) {
 	char *s = NULL;
 
 	// create by given unsigned 32 bit
-	RzILBitVector *bits = rz_il_bv_new_from_ut64(32, 100);
-	RzILBitVector *bits_cmp = rz_il_bv_new(32);
+	RzBitVector *bits = rz_bitvector_new_from_ut64(32, 100);
+	RzBitVector *bits_cmp = rz_bitvector_new(32);
 
 	// 100 = 64 + 32 + 4 == 0b 0000 0000 0000 0000 0000 0000 0110 0100
-	rz_il_bv_set(bits_cmp, 2, true);
-	rz_il_bv_set(bits_cmp, 5, true);
-	rz_il_bv_set(bits_cmp, 6, true);
+	rz_bitvector_set(bits_cmp, 2, true);
+	rz_bitvector_set(bits_cmp, 5, true);
+	rz_bitvector_set(bits_cmp, 6, true);
 	mu_assert("new from 32", is_equal_bv(bits, bits_cmp));
 
 	// dup
-	RzILBitVector *bits_dup = rz_il_bv_dup(bits);
+	RzBitVector *bits_dup = rz_bitvector_dup(bits);
 	mu_assert("dup from bits 32", is_equal_bv(bits_dup, bits));
 
-	s = rz_il_bv_as_string(bits);
+	s = rz_bitvector_as_string(bits);
 	mu_assert_streq_free(s, "00000000000000000000000001100100", "string bit value of bv");
 
-	s = rz_il_bv_as_hex_string(bits);
+	s = rz_bitvector_as_hex_string(bits);
 	mu_assert_streq_free(s, "0x00000064", "string hex value of bv");
 
-	rz_il_bv_free(bits);
-	rz_il_bv_free(bits_cmp);
-	rz_il_bv_free(bits_dup);
+	rz_bitvector_free(bits);
+	rz_bitvector_free(bits_cmp);
+	rz_bitvector_free(bits_dup);
 	mu_end;
 }
 
 bool test_rzil_bv_init64(void) {
```

Comment:
It makes sense to separate just bitvector unit tests from the `test_il_definitions.c` into a new C file, with correspondingly named functions.